docs/copied-from-beats/docs/command-reference.asciidoc

```diff
@@ -121,6 +125,98 @@ endif::[]
 
 Also see <<global-flags,Global flags>>.
 
+ifdef::apm-server[]
+[[apikey-command]]
+==== `apikey` command
+
+{apikey-command-short-desc}
+Most operations require the `manage_security` cluster privilege,
+and you must ensure that `apm-server.api_key` or `output.elasticsearch` are configured appropriately.
+If you modify or delete apm privileges, APM Server might reject all requests.
+
+For more information, see <<api-key>>.
+
+*SYNOPSIS*
+
+["source","sh",subs="attributes"]
+----
+{beatname_lc} apikey SUBCOMMAND [FLAGS]
+----
+
+*SUBCOMMANDS*
+
+// tag::api-key-subcommands[]
```

Comment:
We use `api_key` (in the configuration) and `apikey` for the command itself. Could you rename here to also use `apikey-subcommand` for consistency with `apikey-command`?

Reply:
Good idea